build-scripts/config_common.cmake

```diff
@@ -116,6 +116,24 @@ else ()
   unset (LLVM_AVAILABLE_LIBS)
 endif ()
 
+########################################
+## semantic version information
+
+if (NOT DEFINED WAMR_VERSION_MAJOR)
+  set(WAMR_VERSION_MAJOR 1)
```

Comment:
How about adding a space after set? All set expressions in this file are `set (...)`